src/PlantBundle/Entity/Plant.php

```diff
@@ -78,7 +278,6 @@ public function getOwner()
     public function setOwner($owner)
```

Comment:
You should use explicit typing of variables like this. Use: `public function setOwner(User $owner)`
Do not forget to import `AppBundle\Entity\User`